extract_features.py

```diff
@@ -40,9 +40,12 @@ def get_features(one_student_df: pd.DataFrame) -> pd.DataFrame:
     feature_df['duration'] = feature_df.server_time.diff() / 1000
     feature_df['duration_sd'] = feature_df[['duration']].apply(zscore, nan_policy='omit')
 
-    for outcome in ['OK', 'BUG', 'ERROR', 'INITIAL_HINT']:
+    for outcome in ['ERROR', 'INITIAL_HINT']:
         feature_df['assess_' + outcome] = (one_student_df.tutor_outcome == outcome) * 1
```

Comment:
@pnb is the `*1` important here? should we be replicating that for the new Bug and OK columns? I'm assuming not since multiplying by one shouldn't change the data but want to make sure I am not missing anything before I merge the PR

Reply:
Yes, good catch. `*1` is useful to convert the boolean to a number, which *shouldn't* make a difference for loading in Pandas and sklearn purposes, but I've had too many experiences discovering cases where it accidentally does matter (some other library gets involved, or we try something in R, etc.). So I'd prefer all the features to be numeric.